Package.resolved

```diff
@@ -6,26 +6,26 @@
         "repositoryURL": "https://github.com/Quick/Nimble.git",
         "state": {
           "branch": null,
-          "revision": "8023e3980d91b470ad073d6da843b73f2eeb1844",
-          "version": "7.1.2"
+          "revision": "9c1379fdcd58c4f2278aea5e029394ba9a2b8f07",
+          "version": "7.1.3"
```

Comment:
Should these `Package.resolved` changes be in their own commit?

Reply:
done: 115294ca49121811267edd0b33cb3b691810ce8c